chain/gap/find.go

```diff
@@ -16,12 +16,12 @@ type Finder struct {
 	DB                   *storage.Database
```

Comment:
changes here due to unify range flags (--to and --from) to use int64's. mechanical change